### DISCUSSION TITLE
Day 3: Tips and Tricks Digest Email

### DISCUSSION THREAD

**Discussion by Kourtney-Burger** [SAEL-SWFSC/PAM-SI-Mtg-March2024#59] [Polls]
Beyond this workshop we want to continue sharing new [open science tips, tricks, and resources](https://github.com/SAEL-SWFSC/PAM-SI-Mtg-March2024/discussions/categories/openscience-tips-tricks-and-resources). We encourage all of you to routinely share things that have helped you along your open science journey (including anything relating to GitHub, R, and any other open science tools). 

In the comments below, please leave any other ideas/suggestions on how to best share resources with our NMFS PAM team.

**Comment by mcastellot** [comment]
biweekly?
